packages/office-ui-fabric-react/src/components/TextField/MaskedTextField/MaskedTextField.tsx

```diff
@@ -27,10 +27,10 @@ import {
  *  their indices inside the display text, and their corresponding values.
  * Example:
  *  [
- *    { value: '1', displayIndex: 16, format: /[0-9]/ },
- *    { value: '2', displayIndex: 17, format: /[0-9]/ },
- *    { displayIndex: 18, format: /[0-9]/ },
- *    { value: '4', displayIndex: 22, format: /[0-9]/ },
+ *    \{ value: '1', displayIndex: 16, format: /[0-9]/ \},
+ *    \{ value: '2', displayIndex: 17, format: /[0-9]/ \},
+ *    \{ displayIndex: 18, format: /[0-9]/ \},
+ *    \{ value: '4', displayIndex: 22, format: /[0-9]/ \},
  *    ...
  *  ]
```

Comment:
Since this is meant to be a code sample, you could use a Markdown code fence (three backticks) instead.  Example:

```ts
```